### PR TITLE
[FEATURE] add possibility to specify source address

### PR DESCRIPTION
**Pull request by luhhujbb**
This is directly related to my pull request on java client.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[less-awful-ssl "1.0.0"]
-                 [io.riemann/riemann-java-client "0.4.3"]]
+                 [io.riemann/riemann-java-client "0.4.3-LINKFLUENCE"]]
```
Comment (jamtur01) [inline project.clj]: Let's not change the version until we're reading to increment the client.

Comment (luhhujbb) [inline project.clj]: Ok, my bad, I roll-back to 0.4.3.

**Comment by luhhujbb** [comment]
Thanks for merging ! I may be wrong, but I think now we have to update version of java-client in clojure-client.

**Comment by jamtur01** [comment]
We do. I've not finished that yet.

**Comment by luhhujbb** [comment]
Ok thanks !

**Comment by jamtur01** [comment]
Done.

**Pull request merged**